src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -8,6 +8,9 @@
 
 #include <LibraryResources.h>
 
+#include <Windows.h>
```

Comment:
I'm pretty sure these should actually already _both_ be included in `LibraryIncludes` in `pch.h`, so they shouldn't be needed here.

Reply:
Yeah `pch.h` was including them. Committed the fix to this.